<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1161</width>
      <height>591</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons8-calendrier-96.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/icons8-calendrier-96.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">urlimage: url(:/2.png);
background-color: rgb(193, 193, 193);
background-color: rgb(194, 194, 194);
background-image: url(:/new/prefix1/583453e9474c5 - Copie.jpg);
background-color: rgb(206, 206, 206);
image: url(:/2.png);
image: url(:/2.png);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>FACTURE</string>
     </attribute>
     <widget class="QLabel" name="filtrer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{

color: #1C2951;
 border-style: outset;
     font: bold 16px;
     min-width: 8em;
     padding: 6px;


}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline;&quot;&gt;Filtrer par Client:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="num_F">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>130</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                                                        Num√©ro facture :</string>
      </property>
     </widget>
     <widget class="QLabel" name="montant">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                                      Montant :</string>
      </property>
     </widget>
     <widget class="QLabel" name="type">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}

</string>
      </property>
      <property name="text">
       <string>                                             Client:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filtrer_client">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_fact">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>135</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QTableView" name="table_fac">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>541</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="montant_fac">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>250</y>
        <width>135</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color:#1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_client">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>360</y>
        <width>135</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color:#1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
QComboBox {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_prenom_client">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>CLIENT</string>
     </attribute>
     <widget class="QLabel" name="recherche">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>20</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{

color: #1C2951;
 border-style: outset;
     font: bold 16px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline;&quot;&gt;Recherche par :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_C">
      <property name="geometry">
       <rect>
        <x>-70</x>
        <y>90</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                                                   Identifiant_client :</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                          Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="prenom">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                               Prenom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="date_naisance">
      <property name="geometry">
       <rect>
        <x>-80</x>
        <y>370</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
image: url(:/new/prefix1/2.png);


color: #1C2951;
 border-style: outset;
     font: bold 14px;
     min-width: 8em;
     padding: 6px;


}
</string>
      </property>
      <property name="text">
       <string>                                                   Date de naissance :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_client">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>135</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }

</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_client">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>135</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color:#1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_client">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>135</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QTableView" name="table_client">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>551</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche_client">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>21</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QLineEdit {
     background-color:#DCDCDC;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: #1C2951;
     font: bold 14px;
     min-width: 7em;
     padding: 6px;
}
 QLineEdit {
     image: url(myindicator.png);
 }
</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_n_client">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="test_client">
      <property name="geometry">
       <rect>
        <x>465</x>
        <y>129</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127)</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ajouter_fac"/>
   <addaction name="modifier_fac"/>
   <addaction name="trouver_fac"/>
   <addaction name="suprimer_fac"/>
   <addaction name="stat_facture"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127)</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="trouver_client"/>
   <addaction name="modifier_client"/>
   <addaction name="ajouter_client"/>
   <addaction name="supprimer_client"/>
  </widget>
  <action name="ajouter_client">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/ajouter.png</normaloff>:/new/prefix1/ajouter.png</iconset>
   </property>
   <property name="text">
    <string>ajouter</string>
   </property>
  </action>
  <action name="trouver_client">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/trouver.png</normaloff>:/new/prefix1/trouver.png</iconset>
   </property>
   <property name="text">
    <string>trouver</string>
   </property>
  </action>
  <action name="supprimer_client">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/supprimer.png</normaloff>:/new/prefix1/supprimer.png</iconset>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </action>
  <action name="modifier_client">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/modifier.jpg</normaloff>:/new/prefix1/modifier.jpg</iconset>
   </property>
   <property name="text">
    <string>mod</string>
   </property>
  </action>
  <action name="suprimer_fac">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/icons8-effacer-26.png</normaloff>:/new/prefix1/icons8-effacer-26.png</iconset>
   </property>
   <property name="text">
    <string>sup</string>
   </property>
  </action>
  <action name="trouver_fac">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/icons8-chercher-64.png</normaloff>:/new/prefix1/icons8-chercher-64.png</iconset>
   </property>
   <property name="text">
    <string>trouv</string>
   </property>
  </action>
  <action name="ajouter_fac">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/icons8-plus-48.png</normaloff>:/new/prefix1/icons8-plus-48.png</iconset>
   </property>
   <property name="text">
    <string>ajou</string>
   </property>
  </action>
  <action name="modifier_fac">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/icons8-modifier-48.png</normaloff>:/new/prefix1/icons8-modifier-48.png</iconset>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </action>
  <action name="stat_facture">
   <property name="icon">
    <iconset resource="files/resource.qrc">
     <normaloff>:/new/prefix1/analystic__chart__report__pie__graph__statics-512.png</normaloff>:/new/prefix1/analystic__chart__report__pie__graph__statics-512.png</iconset>
   </property>
   <property name="text">
    <string>stat</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="files/resource.qrc"/>
  <include location="files/resource.qrc"/>
  <include location="files/resource.qrc"/>
 </resources>
 <connections/>
</ui>
